src/main/java/frc/robot/LEDs/LEDIdleCommand.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.Util.LEDs;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Loader.Intake;
+import frc.robot.Shooter.FiringPins;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDAnimation;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDManager;
+
+public class LEDIdleCommand extends CommandBase {
+    protected Intake intake;
+    protected FiringPins firingPins;
+
+    protected LEDAnimation noBallsAnim = LEDManager.STRIP0.fadeAnimation(1,
+        30, 
+        Color.kYellow,
+        Color.kWhite
+    );
+    protected LEDAnimation oneBallAnim = LEDManager.STRIP0.gradientAnimation(1, 
+        Color.kRed,
+        Color.kOrangeRed,
+        Color.kOrange
+    );
+    protected LEDAnimation twoBallsAnim = LEDManager.STRIP0.gradientAnimation(1, 
+        Color.kBlue,
+        Color.kBlueViolet,
+        Color.kPurple
+    );
+
+    // protected LEDAnimation noBallsAnim = LEDManager.STRIP0.gradientAnimation(1, 
+    //     Color.kRed,
+    //     Color.kOrangeRed,
+    //     Color.kOrange
+    // );
+    // protected LEDAnimation twoBallsAnim = LEDManager.STRIP0.gradientAnimation(1, 
+    //     Color.kBlue,
+    //     Color.kBlueViolet,
+    //     Color.kPurple
+    // );
+    // protected LEDAnimation oneBallAnim = LEDAnimation.transposeBlinking(0.04, 
+    //     twoBallsAnim, 
+    //     noBallsAnim
+    // );
+
+    public LEDIdleCommand(LEDSubsystem ledSubsystem, Intake intake, FiringPins firingPins) {
```

Comment:
These Subsystems Don't exist in this years robot.

Reply:
removed irrelevant subsystems and left as a shell command